src/sidebar/services/groups.js

```diff
@@ -77,7 +77,9 @@ function groups(
 
     // If the main document URL has no groups associated with it, always show
     // the "Public" group.
-    const pageHasAssociatedGroups = groups.some(g => g.id !== '__world__');
+    const pageHasAssociatedGroups = groups.some(
```

Comment:
When you say "asynchronously" in the commit message I think you mean "concurrently".